plugins/out_stackdriver/stackdriver.c

```diff
@@ -1405,6 +1405,23 @@ static int get_severity_level(severity_t * s, const msgpack_object * o,
     return -1;
 }
 
+static int get_trace_sampled(int * b, const msgpack_object * o,
```

Comment:
Please use replace these for names that clearly convey what those arguments are.

Reply:
This is something we need to fix for all of these static helper functions, looks like Andre was following our (not good) convention here. We can start by just fixing this signature and I'll add it to my backlog to fix the others. 

Perhaps we could call them `int *trace_sampled_value` and `const msgpack_object *obj`?

Reply:
Sure, that  makes sense.

Reply:
done :)